week2/2-thursday/homework/exercise8/answer.js

```diff
@@ -0,0 +1,22 @@
+// Sort an array 2
+var people = [
+    'Dom',
+    'Lyn',
+    'Kirk',
+    'Autumn',
+    'Trista',
+    'Jesslyn',
+    'Kevin',
+    'John',
+    'Eli',
+    'Juan',
+    'Robert',
+    'Keyur',
+    'Jason',
+    'Che',
+    'Ben'
+  ];
+
+var nameLen = people.sort(function(el, nextel) {
+    return nextel.length - el.length;
+});
```

Comment:
When I run this, i get the longest names first. How would you reverse the order?